routes/rooms.go

```diff
@@ -85,8 +110,8 @@ func CreateEmergencyRoom(ctx *gin.Context) {
 	}
 
 	resp := GetSingleERResponse{
-		EmergencyRoom: er,
-		Departments:   models.DepartmentsToBases(er.Departments),
+		EmergencyRoomBase: er.EmergencyRoomBase,
```

Comment:
The client needs the primary key in order to be capable to operate on the emergency room.